Add content usage guidelines and clarify music licensing (#3660)
ppy/osu-wiki: wiki/Rules/Content_Usage_Guidelines/en.md

# Content Usage Guidelines

> A user should have permission from relevant copyright holders for audio/visual/gameplay assets included in their beatmap submissions. Without permission, the beatmap is at risk of being removed and the creator is at risk of being denied access to the submission system as per the [osu! copyright policy](/wiki/legal/copyright).

Reviewer: ```suggestion
As a mapper, you should have permission from relevant copyright holders for audio/visual/gameplay assets included in their beatmap submissions. Without permission, the beatmap is at risk of being removed and the creator is at risk of being denied access to the submission system as per the [osu! copyright policy](/wiki/legal/copyright).
```

I think we should address as if a mapper is coming to this page, to make it more personal.

To ensure a user's account remains in good standing, it is recommended to create beatmaps using assets that are any of the following:

- legally cleared for use in osu!
- permitted for use by their creator(s)
- copyright-free

## Audio

osu! provides its userbase a library of licensed songs via the [Featured Artist](/wiki/Featured_Artists) program. Each song on this listing is free for use within osu! and has no risk of removal.

When looking for songs outside of osu!'s featured artist catalog, check whether or not its creator has explicitly stated usage terms. This information can often be found in description sections of an artist's music release platforms (e.g. Bandcamp, Soundcloud, YouTube) or their personal website. If a song is licensed under a [Creative Commons License](https://creativecommons.org/licenses/by-nc-sa/3.0/) or is available for non-commercial use, it is safe to create and submit a beatmap for it.

> If you want to create a beatmap for a song and its usage rights are not clear, it is recommended to reach out to its artist for permission. Most artists have contact methods labeled on their music release platforms, personal websites, and social media.

Reviewer: ```suggestion
If you want to create a beatmap for a song and its usage rights are not clear, it is recommended to reach out to its artist for permission. Most artists have contact methods labeled on their music release platforms, personal websites, and social media.

This may feel like a large hurdle, but we've seen mappers have pretty high success and get artists' approval just by taking the courage to ask before using content without permission. It may help to mention that osu! is a free no-ad game, and you can link them to our [licensing](/wiki/Legal/Music_Licensing) page to give them a better idea of how content is consumed.
```

Mention the licensing page and try and encourage mappers to reach out more.

## Visual

Visual assets include a beatmap's background/storyboard/video and follow the same usage requirements as audio assets.

As with music, it is recommended to seek out an artist's input on usage terms and to ask for permission when those terms are unclear. For example, if you're interested in using a background from an artist on pixiv, locate their profile for usage terms or send them a message asking for permission.

Many image-hosting websites have search filters for Creative Commons Licensing or non-commercial use. Some websites that mappers have frequently used to find their copyright-free backgrounds include:

- [pixabay](https://pixabay.com/)
- [Unsplash](https://unsplash.com/)
- [Pexels](https://www.pexels.com/)

If you've received a musician's permission to use their song, their album artwork or any other official artwork for the artist is also likely viable for use as a background/storyboard. Again however, ask for permission if this is not clear.

## Gameplay

> When submitting a beatmap, you should have permission for gameplay-related assets such as [guest difficulties](/wiki/Beatmaps/Beatmapsets/Guest_difficulty) or charts from other rhythm games.

Reviewer: ```suggestion
When submitting a beatmap, you should have permission for gameplay-related assets such as [guest difficulties](/wiki/Beatmaps/Beatmapsets/Guest_difficulty) or charts from other rhythm games.

We do not condone submissions featuring gameplay (hit objects) matching that of other games, as this is not respectful to the rhythm game community.
```

I think it might be good to explicitly state this

> While it can sometimes be difficult to reach out to audio/visual asset creators, mappers are often open to others uploading and crediting their difficulties! Note that plagiarism of gameplay content is also unacceptable, so any difficulties not created solely by you should be labeled accordingly.

Reviewer: Needs a horizontal break above it or something? Feels like an outro paragraph which may read badly against the previous header.
